### PR TITLE
Add `aria-pressed` to all eye/toggle visibility buttons

### LINKED ISSUE #1442: A11y: Toggle Visibility buttons don't expose their state to assistive technologies
### Steps To Reproduce

1. Use a screen reader
2. Navigate to a screen/view with an "eye" icon to toggle visibility
3. Set focus to the control
4. Toggle the control (and if needed, then navigate back to the control, if focus was shifted)

### Expected Result

When focus is on the control, it should announce the state (whether the toggle is pressed or not pressed, i.e. the related password field is visible or not visible).

### Actual Result

Apart from "Toggle Visibility", nothing else is exposed to assistive technologies. Screen reader users have no information to know whether their password is now visible or not.

### Screenshots or Videos

screenshot showing a login screen, with focus on a toggle visibility toggle; NVDA speech viewer is visible, showing that the control is just announced as "toggle visibility"

### Additional Context

See https://github.com/bitwarden/browser/pull/2232

### Operating System

Windows

### Operating System Version

10

### Installation method

Direct Download (from bitwarden.com)

### Build Version

Version 1.32.1 Shell 16.0.7 Renderer 96.0.4664.110 Node 16.9.1 Architecture x64

### PR DESCRIPTION
**Pull request by patrickhlauke**
Closes #1442

## Type of change

- [x] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective

Adds `aria-pressed` to eye/toggle visibility buttons, so they expose their state to assistive technologies

## Code changes

- adds `aria-pressed`, with a value dependent on the state of the control

## Screenshot

Video recording showing the current experience with a screenreader, compared to the experience after this PR is applied - using NVDA on Windows, with the Speech Viewer visible. Note how currently the control is just announced as "Toggle Visibility button", with no indication of state, and how after this PR it announced that it's an actual toggle button, and gives indication whether it's "pressed" or "not pressed"

https://user-images.githubusercontent.com/895831/160274481-4481d045-361f-4989-bbb8-7898238a7840.mp4

## Testing requirements

- test using a screen reader (e.g. NVDA on Windows)

## Before you submit

- [x] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

**Comment by patrickhlauke** [comment]
git push --set-upstream origin patrickhlauke-issue1442

File: src/app/vault/add-edit-custom-fields.component.html [hunk 1]
```diff
@@ -85,6 +85,7 @@
             appBlurClick
             role="button"
             appA11yTitle="{{ 'toggleVisibility' | i18n }}"
+            [attr.aria-pressed]="showValue"
```
Comment (djsmith85) [inline src/app/vault/add-edit-custom-fields.component.html:88]: PR is looking great. Just one minor thing, I discovered during testing. The button would not announce it's state.

```suggestion
            [attr.aria-pressed]="f.showValue"
```

**Review by djsmith85 — CHANGES_REQUESTED** [review]
@patrickhlauke Thanks again, for this great contribution.

Just one minor thing and then we can get this merged.

File: src/app/vault/add-edit-custom-fields.component.html (hunk 1 above)
Comment (patrickhlauke) [reply src/app/vault/add-edit-custom-fields.component.html]: ah, good catch. corrected and tested, now works as it should

**Review by djsmith85 — APPROVED** [review]

**Pull request merged**